src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/StaffRefDataServiceImpl.java

```diff
@@ -486,6 +585,373 @@ public List<UserType> fetchUserTypes() {
     public List<RoleType> getJobTitles() {
         return roleTypeRepository.findAll();
     }
+
+    @Override
+    @SuppressWarnings("unchecked")
+    public StaffProfileCreationResponse updateStaffProfile(StaffProfileCreationRequest profileRequest) {
+
+        log.info("{}:: processStaffProfileCreation starts::",
```

Comment:
Log message should be `update` instead of `creation`

Reply:
done